lib/components/chat_history/decorated_event.dart

```diff
@@ -13,14 +13,13 @@ class DecoratedEventWidget extends StatelessWidget {
   final EdgeInsets padding;
 
   const DecoratedEventWidget._(
-      {Key? key,
+      {super.key,
```

Comment:
Is this what `dart fmt` spits out? It's fine if so, but a little surprised.

Reply:
Yeah dart format nor the analyzer throw me any errors at all, if I do try and correct it to the previous way the latest analyzer warns me to change it again.